### ISSUE TITLE
FormActionType - Confirmation does not work with Closure

### ISSUE THREAD

**Issue by jc-beyer-tqgg** [Kreyu/data-table-bundle#139]
Hey everyone !

The docs say that confirmation can be set to  bool, array or callable.
But when i pass a Closure, i get an error rendering the datatable. It works for href etc. in the ButtonType.

```            
->addRowAction('delete', FormActionType::class, [
                'label' => 'Delete',
                'button_attr' => [
                    'class' => 'btn btn-danger btn-sm',
                ],
                'confirmation' => function(Entity $entity): array {
                    return [
                        'title' => 'Delete',
                        'message' => sprintf(
                            'Are you sure you want to delete Channel: [%s]?',
                            $entity->getChannel()->getName()
                        ),
                    ];
                },
```
                
Is this a bug, wrong doc or am I doing something wrong ? 

Thanks in advance !

**Comment by Kreyu** [comment]
Hey @jc-beyer-tqgg, thanks for the report. There was a bug and I just pushed a fix to version 0.24.3.

Please note that [options allowed in the `confirmation` array](https://data-table-bundle.swroblewski.pl/docs/components/actions.html#adding-action-confirmation) (including array returned from callable) are:

- `translation_domain`
- `label_title` (you'll probably want to use this option instead of `title`)
- `label_description` (you'll probably want to use this option instead of `message`)
- `label_confirm`
- `label_cancel`
- `type`

**Comment by jc-beyer-tqgg** [comment]
@Kreyu Thanks for the super fast fix ! Really appriciate your work here
